azure/posix-sdist.yml

```diff
@@ -0,0 +1,128 @@
+jobs:
+  - job: Linux_Source_Dist
+    pool:
+      vmImage: ubuntu-16.04
+    variables:
+      NIGHTLY_BUILD_COMMIT: "master"
+      NIGHTLY_BUILD: "true"
+      JUNITXML: "test-data.xml"
+      TEST_DIR: "tmp_for_test"
+      TEST_VENV: "test_env"
+      MB_PYTHON_VERSION: "3.8"
+    steps:
+      - checkout: self
+        submodules: true
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: $(MB_PYTHON_VERSION)
+        displayName: Set python version
+      - bash: |
+          set -e
+          SKIP_BUILD="false"
+          if [ "$BUILD_REASON" == "Schedule" ]; then
+            BUILD_COMMIT=$NIGHTLY_BUILD_COMMIT
+            if [ "$NIGHTLY_BUILD" != "true" ]; then
+              SKIP_BUILD="true"
+            fi
+          fi
+
+          echo "Building scikit-learn@$BUILD_COMMIT"
+          echo "##vso[task.setvariable variable=BUILD_COMMIT]$BUILD_COMMIT"
+          echo "##vso[task.setvariable variable=SKIP_BUILD]$SKIP_BUILD"
+        displayName: Define build env variables
+
+      - bash: |
+          set -e
+
+          python -m venv build_env
+          source build_env/bin/activate
+          python -m pip install numpy scipy Cython
```

Comment:
we don't really need cython to create the sdist though. But our setup.py forces it as it is now.

Reply:
Yup. I added a comment regarding why this is needed. (for now)